src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -120,10 +120,11 @@ private function reset(): void
      */
     private function storeEntityToPublish($entity, string $property): void
     {
-        $resourceClass = $this->getObjectClass($entity);
-        if (!$this->resourceClassResolver->isResourceClass($resourceClass)) {
+        $class = $this->getObjectClass($entity);
+        if (!$this->resourceClassResolver->isResourceClass($class)) {
             return;
         }
+        $resourceClass = $this->resourceClassResolver->getResourceClass($entity);
```

Comment:
Shouldn't we add this call directly in the trait, to prevent further similar issues?

Reply:
I'm not sure it'd be a good fit. Do you have any suggestions? There are cases where we want the exception to be thrown if it's not a resource class, but in places like this one, we just want to do nothing.